src/sidebar/util/time.js

```diff
@@ -145,7 +146,7 @@ const BREAKPOINTS = [
  *
  * @param {Date} date - The date to consider as the timestamp to format.
  * @param {Date} now - The date to consider as the current time.
- * @return {breakpoint|null} An object that describes how to format the date or
+ * @return {Object} An object that describes how to format the date or
```

Comment:
Should this be `{Object|null}` or is the text comment wrong?

Reply:
The direct answer to your question is that it seems that TS considers `null` to be a type of `Object`. I changed this to return a more specific `Breakpoint` type and in the process realized that `getBreakpoint` never actually returns `null` because the last item in the `BREAKPOINTS` list will always match. I reworked this code to make it more obvious to both a human reader and TS that this is the case.